### PR TITLE
Add basic examples to the appengine repl

### PR DESCRIPTION
**Pull request by jnthntatum**
Add basic examples to the appengine repl

- Add examples covering basic CEL features and extensions
- Add evaluate button to repl console

### CODE REVIEW

**Review by TristonianJones — DISMISSED** [review]

File: repl/appengine/web/src/app/reference_panel/reference-panel-component.html [hunk 1]
```diff
@@ -16,8 +16,46 @@
...
 <h1>References</h1>
 
-<p>See <a href="https://github.com/google/cel-spec">github.com/google/cel-spec</a> for CEL language
-    overview. </p>
-<p>See
-    <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
-    for REPL syntax guide.</p>
\ No newline at end of file
+<mat-accordion>
+    <mat-expansion-panel>
```
Comment (l46kok) [inline repl/appengine/web/src/app/reference_panel/reference-panel-component.html]: Expansion panels unfortunately won't push the height of its parent container when expanded. So you end up seeing a cut-off content with a scrollbar when the container isn't tall enough to display the expanded content.

You could either change the primary container to something that supports fullscreen (mat-sidenav-container) or just set the viewport heights in your container to be full (not standard angular practice, but it works):

```
.content {
    display: block;
    margin-top: 5vh;
    width: 100%;
    height: 95vh;
}
```

File: repl/appengine/web/src/app/reference_panel/reference-panel-component.html [hunk 2]
```diff
@@ -16,8 +16,46 @@
...
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+            REPL Syntax
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <p>See <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
+            for REPL syntax guide.</p>
+    </mat-expansion-panel>
+    <mat-expansion-panel>
```
Comment (l46kok) [inline repl/appengine/web/src/app/reference_panel/reference-panel-component.html]: Consider expanding the panels by default
```suggestion
    <mat-expansion-panel [expanded]="true">
```

File: repl/appengine/web/src/app/reference_panel/reference-panel-component.html [hunk 3]
```diff
@@ -16,8 +16,46 @@
 
 <h1>References</h1>
 
-<p>See <a href="https://github.com/google/cel-spec">github.com/google/cel-spec</a> for CEL language
-    overview. </p>
-<p>See
-    <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
-    for REPL syntax guide.</p>
\ No newline at end of file
+<mat-accordion>
```
Comment (l46kok) [inline repl/appengine/web/src/app/reference_panel/reference-panel-component.html]: Consider allowing independent expansions:

```suggestion
<mat-accordion multi>
```

File: repl/appengine/web/src/app/reference_panel/reference-panel-component.html (hunk 1 above)
Comment (jnthntatum) [reply repl/appengine/web/src/app/reference_panel/reference-panel-component.html]: Done: I had to restructure the top level app a bit but should behave more like a full screen page.

**Review by l46kok — APPROVED** [review]

**Pull request merged**